java/maven.hints/nbproject/project.properties

```diff
@@ -16,10 +16,10 @@
 # under the License.
 
 is.eager=true
-javac.source=1.8
+javac.source=11
+javac.target=11
 #javac.compilerargs=-Xlint -Xlint:-serial
 
 test.config.stableBTD.includes=**/*Test.class
-requires.nb.javac=true
```

Comment:
the build puts nb-javac into the boot classpath which makes it incompatible with the `release` option (which is set by the ant task automatically). Luckily this module doesn't need nb-javac.